lib/stat_tracker.rb

```diff
@@ -131,6 +131,28 @@ def total_games_per_team_for(season_id)
     end
   end
 
+  def games_won_per_team 
+    @game_teams.reduce(Hash.new(0)) do |result, game_team| 
+      if game_team.result == "WIN" 
+        result[game_team.team_id] += 1 
+      else 
+        result[game_team.team_id] += 0
+      end
+      result
+    end
+  end
+
+  def games_per_team 
+    @game_teams.reduce(Hash.new(0)) do |result, game_team|
+      result[game_team.team_id] += 1 
+      result
+    end
+  end
```

Comment:
We already have a method that does this! `total_games_per_team`

Reply:
Whoops! Thank you. I removed this duplicate helper and updated the win % per team method accordingly.